examples/subscription/mqemitter-mongodb-subscription.js

```diff
@@ -0,0 +1,120 @@
+const GQL = require('fastify-gql')
+const Fastify = require('fastify')
+const mongoose = require('mongoose')
```

Comment:
I would prefer to not use mongoose in an example.

Reply:
done, I was trying an example with a different place where emit events but I understand your point